test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -304,8 +304,10 @@ def _run_tag_failure_IMDSv1_disabled(image_uri, ec2_client, ec2_instance, ec2_co
         ec2_client.delete_tags(Resources=[ec2_instance_id], Tags=[{"Key": expected_tag_key}])
 
     # Disable access to EC2 instance metadata
+    ec2_connection.run(f"sudo apt-get update -y")
+    ec2_connection.run(f"sudo apt-get install -y net-tools")
     ec2_connection.run(f"sudo route add -host 169.254.169.254 reject")
-
+    
```

Comment:
Please remove this extra whitespace. I'd recommend running `black -l 100` on all the files modified in this PR, to allow the code-formatting check to pass on this PR.

Reply:
done

Reply:
will add other file formatting as well